index.js

```diff
@@ -12,10 +13,25 @@ import dbg from 'debug'
 const debug = dbg('jsolines')
 
 /**
- * Create a JSON isoline. Surface is a (possibly typed) array, width and height are its width and height, and cutoff is the cutoff.
- * It is possible to disable linear interpolation for debug purposes by passing interpolation: false
+ * Create a JSON isoline. Surface is a (possibly typed) array, width and height
+ * are its width and height, and cutoff is the cutoff. It is possible to disable
+ * linear interpolation for debug purposes by passing interpolation: false
  */
-export default function jsolines ({surface, width, height, cutoff, project, interpolation = true}) {
+export default function jsolines ({
+  surface,
+  width,
+  height,
+  cutoff,
+  project,
+  interpolation = true
+}: {
+  surface: Uint8Array,
+  width: number,
+  height: number,
+  cutoff: number,
+  project: Function,
```

Comment:
[The Function type is unsafe and should be avoided](https://flow.org/en/docs/types/functions/#toc-function-type).

You could rewrite as follows:

```js
type CoordinateArray = [number, number]

...
  interpolation = true
}: {
  surface: Uint8Array,
  width: number,
  height: number,
  cutoff: number,
  project: CoordinateArray => CoordinateArray,
  interpolation: boolean
}) {
```